update_reminder.sh

```diff
@@ -1,10 +1,36 @@
 #!/bin/bash
 MARKT_LAST_CHANGE=$(wget -q -O- https://wo-ist-markt.de/cities/berlin.json | sed -n 's/.*aktualisiert am \(.*\)",/\1/p')
-BERLIN_LAST_CHANGE=$(wget -q -O- https://daten.berlin.de/datensaetze/berliner-wochen-und-tr%C3%B6delm%C3%A4rkte | sed -n '/.*Aktualisiert.*/,$p' | sed -e '/.*end.*/,$d' | sed -n 's/.*<span .*>\(.*\)<\/span.*/\1/p')
+BERLIN_HEADER_AND_SITE=$(wget --server-response -q -O- https://daten.berlin.de/datensaetze/berliner-wochen-und-tr%C3%B6delm%C3%A4rkte  2>&1  )
 
+#Check Response is 200 (fix https://github.com/wo-ist-markt/wo-ist-markt-berlin-update-reminder/issues/7 )
```

Comment:
"Check Response is 200" should be "Check HTTP status code is 200". But if `BERLIN_STATUS` is named `BERLIN_HTTP_STATUS_CODE` it becomes self documenting.

Reply:
Done